### ISSUE TITLE
Status Screens are Empty for Older Time Ranges

### ISSUE THREAD

**Issue by bsislow** [dalibo/temboard#875]
certain time ranges work on status screens - such as the last 30 minutes.

however, if i extend the range back to 7 days for example, the charts are completely empty:

![image](https://user-images.githubusercontent.com/30243584/97441618-ea7cc900-18f6-11eb-96be-0d5340a43fb1.png)

i'm pretty sure this is related - from the temboard UI log (note i removed sensitive information):

```
2020-10-28 08:20:32,000 temboardui[30093]: [web] ERROR:                                                              ^
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR: Unhandled Error:
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR: Traceback (most recent call last):
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:   File "/usr/lib/python2.7/site-packages/temboardui/web.py", line 227, in error_middleware
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:     return func(request, *args)
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:   File "/usr/lib/python2.7/site-packages/temboardui/web.py", line 408, in user_instance_middleware
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:     return func(request, address, port, *args)
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:   File "/usr/lib/python2.7/site-packages/temboardui/web.py", line 267, in instance_middleware
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:     return callable_(request, *args)
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:   File "/usr/lib/python2.7/site-packages/temboardui/web.py", line 431, in user_middleware
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:     return func(request, *args)
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:   File "/usr/lib/python2.7/site-packages/temboardui/plugins/monitoring/handlers/monitoring.py", line 164, in data_metric
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:     key=key,
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:   File "/usr/lib/python2.7/site-packages/temboardui/plugins/monitoring/chartdata.py", line 872, in get_metric_data_csv
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:     data_buffer)
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR: UndefinedColumn: column "key" does not exist
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR: LINE 6: ...020-10-28T13:20:54.184000+00:00'::timestamptz AND key = 'rep...
2020-10-28 08:20:32,002 temboardui[30093]: [web] ERROR:                                                              ^
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR: Request failed: 500 column "key" does not exist
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR: LINE 6: ...020-10-28T13:20:54.184000+00:00'::timestamptz AND key = 'tem...
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR:                                                              ^
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR: .
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR: Request failed: 500 column "key" does not exist
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR: LINE 6: ...020-10-28T13:20:54.184000+00:00'::timestamptz AND key = 'neg...
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR:                                                              ^
2020-10-28 08:20:32,003 temboardui[30093]: [web] ERROR: .
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR: Request failed: 500 column "key" does not exist
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR: LINE 6: ...020-10-28T13:20:54.184000+00:00'::timestamptz AND key = 'pos...
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR:                                                              ^
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR: .
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR: Request failed: 500 column "key" does not exist
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR: LINE 6: ...020-10-28T13:20:54.184000+00:00'::timestamptz AND key = 'rep...
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR:                                                              ^
2020-10-28 08:20:32,004 temboardui[30093]: [web] ERROR: .
2020-10-28 08:20:32,009 temboardui[30093]: [access] ERROR: 500 GET /server/xxxxxx/xxxxxx/monitoring/data/rollback_db?key=postgres&start=2020-10-26T13%3A20%3A54.183Z&end=2020-10-28T13%3A20%3A54.184Z (10.244.244.145) 42.65ms
2020-10-28 08:20:32,009 temboardui[30093]: [access] ERROR: 500 GET /server/xxxxxx/xxxxxx/monitoring/data/rollback_db?key=template1&start=2020-10-26T13%3A20%3A54.183Z&end=2020-10-28T13%3A20%3A54.184Z (10.244.244.145) 60.57ms
2020-10-28 08:20:32,009 temboardui[30093]: [access] ERROR: 500 GET /server/xxxxxx/xxxxxx/monitoring/data/rollback_db?key=negval&start=2020-10-26T13%3A20%3A54.183Z&end=2020-10-28T13%3A20%3A54.184Z (10.244.244.145) 38.79ms
2020-10-28 08:20:32,010 temboardui[30093]: [access] ERROR: 500 GET /server/xxxxxx/xxxxxx/monitoring/data/rollback_db?key=repmgr&start=2020-10-26T13%3A20%3A54.183Z&end=2020-10-28T13%3A20%3A54.184Z (10.244.244.145) 52.32ms
```

FYI:
```
$ rpm -qa | grep temboard
temboard-agent-7.0a6-1.el7.noarch
temboard-7.2-1.el7.noarch
```

**Comment by pgiraud** [comment]
Duplicate of #888

**Comment by pgiraud** [comment]
Fix shipped with 7.6.
